packages/component/src/Middleware/ActivityGrouping/ui/PartGrouping/CollapsibleGrouping.tsx

```diff
@@ -17,8 +17,8 @@ const { useLocalizer } = hooks;
 const collapsibleGroupingPropsSchema = pipe(
   object({
     children: reactNode(),
-    title: string(),
-    titleClassName: optional(string()),
+    className: optional(string()),
+    header: optional(reactNode()),
```

Comment:
Passing `ReactNode`... will it hurt cacheability? Not sure, but I guess every time the parent render, it could be a new object.